test/fixtures/rest/app/apis/users/posts/answer/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+exports.index = function* () {
```

Comment:
@mattma Forget to comment. I mean `/api/users/3/posts/1/replies/2/answer` is matching to `apis/users/posts/replies/answer.js`. I can retrive the ids from `this.params.parent_ids` which is

```
{
  users: '3',
  posts: '1',
  replies: '2',
}
```